omniscidb/ArrowStorage/ArrowStorageUtils.cpp

```diff
@@ -196,6 +199,8 @@ std::shared_ptr<arrow::ChunkedArray> replaceNullValuesImpl(
   std::shared_ptr<arrow::Array> array;
   if constexpr (std::is_same_v<T, bool>) {
     array = std::make_shared<arrow::Int8Array>(arr->length(), std::move(resultBuf));
+  } else if (arr->type()->id() == arrow::Type::NA) {
```

Comment:
I have a doubt here. What if we have an existing table with int or some other column and try to import NULL column into it? It's supposed to work, but here we always create doubles. I'm not sure if it later matters and this type is used anywhere, but a better way here would be to make arrow type out of the known C type. 
```
using ResultArrowType = typename arrow::CTypeTraits<T>::ArrowType;
using ArrayType = typename arrow::TypeTraits<ResultArrowType>::ArrayType;
```

Reply:
Ok, changed to ArrayType for consistency and have Null -> fp64 conversion at single case. Testscase with appendance of Null column to schema discovered from previous csv with int. 
 I'm not sure if there will be any changes in logic after this fix.